### PR TITLE
Frequently Asked Questions

### PR DESCRIPTION
**Pull request by davidmyriel**
This PR adds an FAQ section to the docsite.
The questions are separated by topic: General, Cloud, Database Tuning, Server Hanging.

### CODE REVIEW

File: qdrant-landing/content/documentation/faq/server-issues.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+The easiest fix would be to set max_retries to a higher value and set the timeout higher as well when you instantiate QdrantClient. However, we also recommend keeping the entire batch size (including vector & payload) under 5MB.
+
+### I have too many vectors and I can’t upload them all to RAM.
+
+Directly writing to disk is a lot faster when doing a huge upload batch.You want to use memmap support. During collection creation, memmap may be enabled on a per-vector basis using the on_disk parameter. This will store vector data directly on disk. Read more about [configuring mmap storage](../../concepts/storage/#configuring-memmap-storage).
+
+### The server stopped responding via HTTP. 
+
+If you indexed a large number of points and there is not response, try debugging. If no information is available in debug.
```
Comment (agourlay) [inline qdrant-landing/content/documentation/faq/server-issues.md:18]: Is this section not finished?

> If no information is available in debug.

What are users supposed to do?

Comment (davidmyriel) [reply qdrant-landing/content/documentation/faq/server-issues.md:18]: Yes this isnt finished. What should we say?

Comment (agourlay) [reply qdrant-landing/content/documentation/faq/server-issues.md:18]: Well in that case, they should contact support with all the necessary information to reproduce the issue.

Comment (davidmyriel) [reply qdrant-landing/content/documentation/faq/server-issues.md]: Added, please approve and ill publish if OK.

**Review by agourlay — APPROVED** [review]
That's a great compilation of recurrent questions :+1:

**Pull request merged**